include/sys/zio.h

```diff
@@ -527,6 +509,10 @@ extern void *zio_data_buf_alloc(size_t size);
 extern void zio_data_buf_free(void *buf, size_t size);
 extern void *zio_buf_alloc_flags(size_t size, int flags);
 
+extern void zio_push_transform(zio_t *zio, void *data, uint64_t size,
+    uint64_t bufsize, zio_transform_func_t *transform);
+extern void zio_pop_transforms(zio_t *zio);
```

Comment:
Is there a reason these functions were made non-static? It doesn't look like we use them anywhere. Maybe this is for the rest of the ABD patches?